demo/presto/node.properties

```diff
@@ -1,3 +1,21 @@
 node.environment=docker
 node.id=ffffffff-ffff-ffff-ffff-ffffffffffff
 node.data-dir=/var/presto/data
+node.id=974b84d2857b
```

Comment:
I'm curious what is this property and what happens when it is repeated like this?